### PR TITLE
[DOC] Fix broken link to cookie secret generation

### PR DESCRIPTION
**Pull request by koriit-kontakt**
## Description
Just fixing a broken link to cookie secret generation.

## Motivation and Context
I was looking for something, I thought I found it but the link was broken. I am 100% sure that in the future this will happen again :D so I'd like to save myself from the trouble.

### CODE REVIEW

File: docs/docs/configuration/auth.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ Whether you are using GitLab.com or self-hosting GitLab, follow [these steps to
 
 If you need projects filtering, add the extra `read_api` scope to your application.
 
-The following config should be set to ensure that the oauth will work properly. To get a cookie secret follow [these steps](https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/configuration/configuration.md#configuration)
+The following config should be set to ensure that the oauth will work properly. To get a cookie secret follow [these steps](https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/docs/configuration/overview.md)
```
Comment (JoelSpeed) [inline docs/docs/configuration/auth.md:166]: Perhaps this link should point to the correct place on the docs site instead of the raw markdown, WDYT?

Comment (koriit-kontakt) [reply docs/docs/configuration/auth.md:166]: That would be the best. The link to blob was there so I didn't want to change it thinking that there might have been a reason behind it. :D

Comment (koriit-kontakt) [reply docs/docs/configuration/auth.md]: Ok, I think this is the proper way of generating site links.

**Review by JoelSpeed — APPROVED** [review]
Just checked this out and it seems to be working as expected! Thanks :D

**Pull request merged**